web/src/components/atoms/Icon/index.tsx

```diff
@@ -1,6 +1,18 @@
-import { HomeOutlined, DownloadOutlined } from "@ant-design/icons";
+import {
+  HomeOutlined,
+  DownloadOutlined,
+  UploadOutlined,
+  FullscreenOutlined,
+  ExclamationCircleTwoTone,
+} from "@ant-design/icons";
 
 export default {
   home: HomeOutlined,
   download: DownloadOutlined,
+  upload: UploadOutlined,
+  fullscreen: FullscreenOutlined,
+};
+
+export const twoTone = {
+  exclamationCircle: ExclamationCircleTwoTone,
```

Comment:
Any reason you split like this?
Also, just so you know icons will be updated once [this PR](https://github.com/reearth/reearth-cms/pull/139) is merged. They'll act same as in Re:Earth from now!

Reply:
Once that PR is merged, please update the Icon component to allow for antd Icon's properties since @Basel-Issmail was leaving it out until needed

Reply:
because in AntD we have three types of icons:
1- Outlined. 2- Filled. 3- TwoTone
I thought that TwoTone icons should not be exported with Outline icons.

Reply:
I temporarily used "ExclamationCircleOutlined" instead of "ExclamationCircleTwoTone" until TwoTone icons become supported in the "Icon" atom component.